src/firefly/java/edu/caltech/ipac/firefly/server/query/ObsCorePackager.java

```diff
@@ -58,10 +67,15 @@ private List<FileGroup> computeFileGroup(DownloadRequest request) throws DataAcc
                 }
                 String access_format = (String) dgDataUrl.get(idx, "access_format");
                 URL url = new URL(access_url);
+
                 File tmpFile = File.createTempFile("obscorepackager-", ".download", ServerContext.getTempWorkDir());
                 FileInfo tmpFileInfo = URLDownload.getDataToFile(url, tmpFile);
                 String contentType = tmpFileInfo.getAttribute(HttpResultInfo.CONTENT_TYPE);
 
+                //todo: replace tmpFile code above with this, or else remove this commented out code
+                //HttpResultInfo r= URLDownload.getDataFromURL(new URL(access_url), null, null, null);
+                //String contentType = r.getContentType();
```

Comment:
FYI @robyww @loitly 
- based on our discussion, I had thought about removing the tmpFile code, as we could just use `access_format` to get the `extension`, but I realized that the `else` case (~line 103) is actually triggered when `access_format` is `null`. So I have left this code and the commented out line here for now, but we can discuss it.